### DISCUSSION TITLE
printf or conio functions with switched VIC bank

### DISCUSSION THREAD

**Discussion by pizza666** [cc65/cc65#1722] [Q&A] [answered]
When switching VIC bank with CIA2.pra ($dd00) and offset with VIC.addr ($d018) how can I tell conio functions to use the remapped screen RAM?

**Comment by mrdudz** [comment, marked as answer]
I don't know if conio is actually respecting this, but try setting $0288 to the highbyte of your screen RAM

**Reply by pizza666** [reply]
It actually does thanks alot!
